File: Day 3/Aayushi-Mittal.cpp
// Write a program to find out the total number of days in a given month.

#include<iostream>
using namespace std;
int main()
{
  string month;
  cout<<"Enter a month name (in lower case) :";
  cin>>month;
  if(month == "january" || month == "march" || month == "may" || month == "july" || month == "august" || month == "october" || month == "december")
    cout<<"\n> "<<month<<" has 31 days"<<endl;
  else if(month == "april" || month == "june" || month == "september" || month == "november")
    cout<<"\n> "<<month<<" has 30 days"<<endl;
  else if(month == "february")
  {
    cout<<"\n> "<<month<<" has 28 or 29 days"<<endl;
    int year;
    cout<<"Enter the current year: ";
    cin>>year;
    // checking for leap year or not.
    (year%400 == 0 || (year%100 != 0 && year%4 == 0)) ? cout<<"\n> "<<year<<" is a leap year. So, "<<month<<" has 29 days." : cout<<"> "<<year<<" is not a leap year. So, "<<month<<" has 28 days.";
  }
  else
    cout<<"> "<<month<<" is invalid.";
}

File: Day 3/Aayushi-Mittal.py
# Write a program to find out the total number of days in a given month.

month = input("Enter a month name (in lower case) : ")
if month == "january" or month == "march" or month == "may" or month == "july" or month == "august" or month == "october" or month == "december":
    print("\n> " + month + " has 31 days")
elif month == "april" or month == "june" or month == "september" or month == "november":
    print("\n> " + month + " has 30 days")
elif month == "february":
    print("\n> " + month + " has 28 or 29 days")
    year = int(input("Enter the current year: "))
    if year%400 == 0 or (year%100 != 0 and year%4 == 0):
        print(f"\n> {year} is a leap year. So, {month} has 29 days.")
    else:
        print(f"\n> {year} is not a leap year. So, {month} has 28 days.")
else:
    print("> " + month + " is invalid.")
